Automattic/jetpack
# Backup: Fix npm script build paths for the plugin

#### Fixes

* Fix monorepo npm script build paths for the Jetpack Backup plugin

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No.

#### Testing instructions:

* The monorepo commands `build-jetpack-backup` and `build-jetpack-backup-concurrently` should now run without issue, even if the `projects/plugins/backup` directory doesn't exist.
